examples/react-native/scripts/remove-resolutions.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+# You'll need to install `jq`: https://stedolan.github.io/jq/
+
+# Removes `@builder.io/sdk-react-native` resolution from the list of resolutions in `package.json``
+echo "$(jq 'del(.resolutions."@builder.io/sdk-react-native")' package.json)" >package.json
```

Comment:
These scripts can hopefully all be replaced by `yarn link <package-name>` commands once this mono-repo moves to yarn workspaces, but right now that won't work